src/main/java/eiteam/esteemedinnovation/tools/ToolsModule.java

```diff
@@ -141,6 +141,8 @@ public void registerItems(RegistryEvent.Register<Item> event) {
         ROTARY_BLADES = setupUpgrade(event, new ItemRotaryBladesUpgrade(), "rotary_blades");
         SIFTER = setupUpgrade(event, new ItemSifterUpgrade(), "sifter");
         BACKHOE = setupUpgrade(event, new ItemBackhoeUpgrade(), "backhoe");
+
+        DrillHeadMaterial.registerDefaults();
```

Comment:
Why was this moved? This must be done after all mod items and OreDict entries (not just EI) have been registered. See: https://github.com/Esteemed-Innovation/Esteemed-Innovation/blob/4230f9e3a89f9175afc720fcc6575300058d16b3/src/main/java/eiteam/esteemedinnovation/tools/steam/upgrades/drillhead/DrillHeadMaterial.java#L126

Reply:
Because it was in the Recipes method. Recipes happens before item registration so it needed to be moved.

Reply:
Ah, I was unaware that the time recipes are set up changed. Hm. It might be better to move this into `ToolsModule#finish()`